File: 1-two_sum/solution.py
# Beats 50% memory
# Beats 50% runtime

# Pretty much default solution

class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        prevValues = []
        for i, j in enumerate(nums):
            neededValue = target - j
            if neededValue in nums and nums.index(neededValue) != i:
                return [i, nums.index(neededValue)]

File: 1-two_sum/solution.js
// Beats 100% in memory

/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    for(i in nums){
        tar = target - nums[i];
        tartar = nums.indexOf(tar)
        if(nums.includes(tar) && i != tartar){
            return [i, tartar];
        }
    }
};

